modules/researchsoftware/slides-software.md

```diff
@@ -108,13 +101,11 @@ These punchcards would be loaded into the computer to instruct which parts of th
 
 ===
 
-<!-- .slide: data-state="standard" -->
+<!-- .slide: data-state="standard center" -->
```

Comment:
We might want to consider, removing this slide with the unsplash image altogether because we have the VS code screenshot in the next slide. I think that is way better and having both seems a bit redundant.

Reply:
Agreed. 
I updated this part a little bit in bf95ea3. Is it better like this, or do you think we should leave this slide out altogether?

Reply:
Yeah, this is great!